public/javascripts/main.js

```diff
@@ -62,3 +62,33 @@ $(document).ready(function() {
     })
 });
 
+$(function () {
+    var dateObj = new Date();
+    var hour = dateObj.getHours();
+    var day = dateObj.getDay();
+
+    if((hour >= 15 && day == 4) || day == 5) { // Starting Thursday 3 PM onwards to Friday
```

Comment:
please this more generic. it should be from the day before starting at 15:00

Reply:
Comment addressed.